### ISSUE TITLE
Screen Mirroring like Airplay for non-touch screen brands like Audi

### ISSUE THREAD

**Issue by ivelliosy** [EthanArbuckle/carplay-cast#65]
Thanks for all the contribution Ethan. Unfortunately, tweak cannot be used on non-touch screen vehicles like Audi's, Mercedes etc. due to joystick control. Can you add a feature similar to "Portal" thing in CarBridge to be able to mirror screen directly? Happy to donate as well. Thanks!

**Comment by devil52573** [comment]
Yes! I'm having this problem too with Mercedes without touch screen.
I suggest to disable the screen showing "Running on carplay screen"
So we can control the apps from our phone.
Thanks @EthanArbuckle
